lib/open_project/costs/engine.rb

```diff
@@ -219,6 +219,14 @@ class Engine < ::Rails::Engine
                      represented.project.costs_enabled?)
              }
 
+      # N.B. in the long term we should have a type like "Currency", but that requires a proper
+      # format and not a string like "10 EUR"
+      schema :overall_costs,
+             type: 'String',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.